assets/sparksql/Dockerfile

```
@@ -0,0 +1,13 @@
+# FROM spark:python3
+
+FROM spark:4.0.0
+
+USER root
```

Hadolint (2.12.0)

[warning] Last USER should not be root

(DL3002)